.github/workflows/release-notes.yml

```
@@ -46,15 +64,34 @@ jobs:
       run: |
         java -jar ${{ steps.download-changelog-cli.outputs.file }} \
         -fc "${{ github.event.inputs.starting_commit }}" \
-        -ex "{\"version\":\"${{ steps.get-version.outputs.version }}\"}" \
+        -ex "{\"version\":\"${{ steps.get-version.outputs.resolved-version }}\"}" \
         -t .github/release_notes_template/template.hbs \
         -hhf .github/release_notes_template/helper.hbs \
-        -of kura/distrib/RELEASE_NOTES.txt
+        -of RELEASE_NOTES.txt
+    
+    - name: Files exist write description
+      id: get-description
+      if: steps.check_files.outputs.exists == 'true' && github.event.inputs.overwrite == false
+      # Only runs if all of the files exist
+      run: |
+          awk 'NR==FNR { desc = (desc == "" ? $0 : desc "\n" $0); next } /\[WIP\]/ && !done { sub(/\[WIP\]/, desc); done=1 } 1' description.txt done=0 RELEASE_NOTES.txt
+          awk 'NR==FNR { desc = (desc == "" ? $0 : desc "\n" $0); next } /\[WIP\]/ && !done { sub(/\[WIP\]/, desc); done=1 } 1' target-env.txt done=0 RELEASE_NOTES.txt
+          awk 'NR==FNR { desc = (desc == "" ? $0 : desc "\n" $0); next } /\[WIP\]/ && !done { sub(/\[WIP\]/, desc); done=1 } 1' known-issues.txt done=0 RELEASE_NOTES.txt
+      shell: bash
+        
+    - name: Files exist clean up
+      id: clean-up-files
+      if: steps.check_files.outputs.exists == 'true' && github.event.inputs.overwrite == false
+      # Only runs if all of the files exist
+      run: |
+        rm description.txt
+        rm target-env.txt
+        rm known-issues.txt
 
     - name: Create Pull Request
       uses: peter-evans/create-pull-request@v4
       with:
-        title: "chore: add Kura ${{ steps.get-version.outputs.version }} release notes"
-        commit-message: "chore: add Kura ${{ steps.get-version.outputs.version }} release notes"
-        body: "Automated changes by _Release Notes automation_ action: add Kura ${{ steps.get-version.outputs.version }} version release notes since commit `${{ github.event.inputs.starting_commit }}` (non-inclusive)"
-        branch-suffix: short-commit-hash
+        title: "chore: add Project ${{ steps.get-version.outputs.resolved-version }} release notes"
+        commit-message: "chore: add Project ${{ steps.get-version.outputs.resolved-version }} release notes"
+        body: "Automated changes by _Release Notes automation_ action: add Project ${{ steps.get-version.outputs.resolved-version }} version release notes since commit `${{ github.event.inputs.starting_commit }}`"
+        branch-suffix: timestamp
```

Mh... maybe it's better to not change this

```suggestion
        title: "chore: add Kura ${{ steps.get-version.outputs.resolved-version }} release notes"
        commit-message: "chore: add Kura ${{ steps.get-version.outputs.resolved-version }} release notes"
        body: "Automated changes by _Release Notes automation_ action: add Kura ${{ steps.get-version.outputs.resolved-version }} version release notes since commit `${{ github.event.inputs.starting_commit }}`"
        branch-suffix: short-commit-hash
```